### DISCUSSION TITLE
How to run code-server and use it with --link but without SSH.

### DISCUSSION THREAD

**Discussion by dedfft** [coder/code-server#4152] [Q&A]
when i launch code-server with --link it works ONLY why i am connected to ssh. But somehow on one of my droplets it works via link and github account, but without ssh connection. How to do the same?

**Comment by lg-nineoneninethreenine** [comment]
So, in your first droplet, it works only on SSH but on the other one, it works without SSH, right? I guess the one that works only on SSH has some change in config or something that makes it run some commands only on SSH. Check your VM config.
